### ISSUE TITLE
Incomplete Sentence in ZopeInterfaceMethod Comment

### ISSUE THREAD

**Issue by mrthankyou** [github/codeql#5650]
[This](https://github.com/github/codeql/blob/2d618d6b928d8b76ac8033b3b63d9bde71caa325/python/ql/src/semmle/python/libraries/Zope.qll#L6) sentence feels incomplete. I'm confused by it and don't understand what it means. 

Is this just me or are others also confused?

**Comment by yoff** [comment]
I agree. From reading the surrounding comments, it should be "that belongs to". But this file should really be modernised.

**Comment by mrthankyou** [comment]
Thank you for mentioning it needs to be modernized. I was looking at that file trying to learn CodeQL.

**Comment by yoff** [comment]
No problem, we still have a few out-dated bits around. A good place to look for example code is `Flask.qll` which me made ready some time ago for this purpose. I think we forgot to promote this when we updated the documentation.
